apps/editor.planx.uk/src/pages/FlowEditor/components/NewSettings/TeamSettings/IntegrationSettings/SubmissionEmails/schema.ts

```diff
@@ -0,0 +1,16 @@
+import { boolean, number,object, string } from "yup";
+
+import { SubmissionEmailFormValues } from "./types";
+
+export const validationSchema = object().shape({
+  teamId: number().required(),
```

Comment:
We don't actually need this to be part of the form - there's no corresponding field for it so formik validation can never pass.

We do need it for the mutation variables, so we can pass it directly in via `getMutationVariables()` (which you're already doing).